src/network/network.cpp

```diff
@@ -1,6 +1,6 @@
 /*
     SlimeVR Code is placed under the MIT license
-    Copyright (c) 2021 Eiren Rain
+    Copyright (c) 2021 Eiren Rain & SlimeVR contributors
```

Comment:
"SlimeVR contributors" is not a legal entity and should not appear in a copyright header. Either put Eiren or remove it completely.

Reply:
Resolving, as I just don't care enough, and because who really does?
I'm also resolving my other nits about this topic, those can be brought up in a different PR